packages/compiler-cli/src/ngtsc/annotations/src/component.ts

```diff
@@ -34,6 +36,7 @@ export interface ComponentHandlerData {
   meta: R3ComponentMetadata;
   parsedTemplate: TmplAstNode[];
   metadataStmt: Statement|null;
+  parseTemplate(options?: ParseTemplateOptions): ParsedTemplate;
```

Comment:
```suggestion
  parseTemplate: (options?: ParseTemplateOptions) => ParsedTemplate;
```

To me, the former implies that `parseTemplate` is a method on `ComponentHandlerData`, whereas the latter more means that the field holds a closure we can call that's otherwise independent of `ComponentHandlerData`.

Reply:
I agree. Interesting; for a while I thought the former was syntactic sugar for the latter, but the latter is much more indicative of a stateful closure. Thanks!